packages/darklang/languageTools/parser.dark

```diff
@@ -229,6 +234,21 @@ module Darklang =
               (WrittenTypes.TypeReference.Builtin.TString node.sourceRange)
               |> Stdlib.Result.Result.Ok
 
+            | text when (Stdlib.String.startsWith text "List") ->
```

Comment:
Not sure if this good, if you have a better way in mind (to match directly against `List`) please let me know.
I've tried updating the grammar to `alias ("List", $.list_type)` , but It didn't seem right

Reply:
You have to wrap that alias in a field to make the grammar.js change have any effect -- mentioned in more detail there.

Reply:
Yes, I remember something about anonymous nodes. I just wasn't sure if it was appropriate to use a field in this case, but this confirms it. Thanks.